src/main/java/seedu/address/logic/commands/ListPersonCommand.java

```diff
@@ -1,24 +1,60 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
 import seedu.address.model.Model;
+import seedu.address.model.person.NameContainsKeywordsPredicate;
 
 /**
- * Lists all persons in the address book to the user.
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ * If no keywords are specified, all persons will be listed.
  */
 public class ListPersonCommand extends Command {
 
     public static final String COMMAND_WORD = "listPerson";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
```

Comment:
"Lists" may be more appropriate than "Finds"

Reply:
There is the next part "and display them as a list..." so I think it should be ok